sub-packages/bionemo-core/src/bionemo/core/data/resources/scdl.yaml

```diff
@@ -5,3 +5,9 @@
   sha256: 7a4237537bf535dfa00301ce8cc7073e0a23d5bc8aa902ad65db9f51b57a6df9 # pragma: allowlist secret
   owner: Polina Binder <pbinder@nvidia.com>
   description: Sample test data for SCDL.
+
+- tag: sample_scdl_feature_ids
+  pbss: "s3://bionemo-ci/test-data/scdl_feat_ids_sample_updated.tar.gz"
```

Comment:
can you add any file that is part of a test or part of the notebook example into ngc? @dorotat-nv can help.